rime_main.c

```diff
@@ -85,23 +170,27 @@ void ibus_rime_start(gboolean full_check) {
   ibus_rime_traits.distribution_code_name = DISTRIBUTION_CODE_NAME;
   ibus_rime_traits.distribution_version = DISTRIBUTION_VERSION;
   ibus_rime_traits.app_name = "ibus";
-  static RIME_MODULE_LIST(ibus_rime_modules, "default", "legacy");
-  ibus_rime_traits.modules = ibus_rime_modules;
+  ibus_rime_traits.modules = NULL;
+
```

Comment:
Please help add `rime_api->setup(&ibus_rime_traits);` here to enable logging

Reply:
No. I got the following error when deploying:
```
F0621 11:33:03.380916  3420 utilities.cc:322] Check failed: !IsGoogleLoggingInitialized() You called InitGoogleLogging() twice!
*** Check failure stack trace: ***
    @     0x7fb2041b0cad  google::LogMessage::Fail()
    @     0x7fb2041b2b13  google::LogMessage::SendToLog()
    @     0x7fb2041b0808  google::LogMessage::Flush()
    @     0x7fb2041b3579  google::LogMessageFatal::~LogMessageFatal()
    @     0x7fb2041b9e42  google::glog_internal_namespace_::InitGoogleLoggingUtilities()
    @     0x55aff0688bef  ibus_rime_start
    @     0x55aff06887ef  ibus_rime_engine_property_activate
    @     0x7fb2052a7c8d  g_closure_invoke
    @     0x7fb2052babbd  (unknown)
    @     0x7fb2052c42be  g_signal_emit_valist
    @     0x7fb2052c497f  g_signal_emit
    @     0x7fb2054ce0a6  (unknown)
    @     0x7fb2053e21e9  (unknown)
    @     0x7fb2051c5dd8  g_main_context_dispatch
    @     0x7fb2051c61c8  (unknown)
    @     0x7fb2051c64c2  g_main_loop_run
    @     0x7fb2054e445c  ibus_main
    @     0x55aff0687695  main
    @     0x7fb204d1c09b  __libc_start_main
    @     0x55aff06876fa  _start
Aborted
```

Reply:
We need to add a one-time `setup()` call before first `initialize()`.

Sorry I mentioned that requirement in a separate comment. It means extra refactoring effort to extract the traits struct and `setup()` out of the recurring `ibus_rime_start()` function.